src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -29,9 +29,7 @@ public class HubEndPoint<THub> : HubEndPoint<THub, IClientProxy> where THub : Hu
 
     public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub : Hub<TClient>
     {
-        private readonly Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>> _callbacks
-            = new Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>>(StringComparer.OrdinalIgnoreCase);
-        private readonly Dictionary<string, Type[]> _paramTypes = new Dictionary<string, Type[]>();
+        private readonly Dictionary<string, HubMethodDescriptor> _methods = new Dictionary<string, HubMethodDescriptor>(StringComparer.OrdinalIgnoreCase);
```

Comment:
They've always been in the past. Also, most things we do in ASP.NET Core are case insensitive in general (except the file system). What's the downside?